src/main/resources/model/instance.json

```diff
@@ -95,6 +95,20 @@
           "index": "keyword"
         }
       }
+    },
+    "publication": {
+      "type": "object",
+      "properties": {
+        "publisher": {
+          "inventorySearchTypes": "publisher",
+          "index": "multilang",
+          "showInResponse": true
+        },
+        "dateOfPublication": {
+          "index": "source",
```

Comment:
What is `source` index type means?

Reply:
it means that value would be stored in elasticsearch, but it would be able for search (ES won't create reverse index for it)